src/shady-data.js

```diff
@@ -10,40 +10,6 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
 
 
 export class ShadyData {
-  constructor() {
```

Comment:
This is just removing fud about object shaping?

Reply:
Yes